config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'post/index'
```

Comment:
* When i ran the `rails routes` command i didn't get the expected results kindly look into it 
![image](https://github.com/monicah79/Renters_ke/assets/52497246/15c59301-cc2c-4be1-8c13-e4e5a810c77f)
![image](https://github.com/monicah79/Renters_ke/assets/52497246/faae9e5f-6254-4c61-ac29-fd1adaaa7b17)

Reply:
Hey here is my screenshoot
![image](https://github.com/monicah79/Renters_ke/assets/107432779/5d6d3823-27f7-4815-874c-08e950e3f0c6)

Reply:
![image](https://github.com/monicah79/Renters_ke/assets/107432779/700a39a3-dff8-4657-8300-2686d2883520)